### ISSUE TITLE
Playing multiple audio tracks simultaneously

### ISSUE THREAD

**Issue by Tosh0kan** [mpvnet-player/mpv.net#524]
Is it possible to do something like loading a video file and them load an external audio track and have both of the video file and the external audio play their audio tracks at the same time? I was wondering this mainly because I wanna watch movies with commentary tracks, so being able to control their volumes independently would also be best.

**Comment by Cole-Graham** [comment]
Yes this would be a great feature, I use it to browse recordings of game footage which have separate audio tracks for game/desktop/voice/spotify,etc.

**Comment by stax76** [comment]
What I found on this topic:

https://github.com/mpv-player/mpv/issues/10554

**Comment by Tosh0kan** [comment]
it only took 18 months to get an answer lmfao
